transforms/contains-to-includes/index.js

```diff
@@ -0,0 +1,21 @@
+const { getParser } = require('codemod-cli').jscodeshift;
+
+module.exports = function transformer(file, api) {
+  const j = getParser(api);
+
+  return j(file.source)
+    .find(j.CallExpression, {
+      callee: {
+        type: 'MemberExpression',
```

Comment:
For my learning and understanding, what is a `MemberExpression`? I tried to find decent documentation about `jscodeshift` but had no luck. 

Any resource/s you could recommend?

Reply:
Yeah so, docs on this stuff are a little tricky to find, https://babeljs.io/docs/en/babel-types#memberexpression this shows some information on types